src/SignalR/common/Http.Connections/src/Internal/Transports/WebSocketsServerTransport.cs

```diff
@@ -288,7 +288,7 @@ private async Task StartSending(WebSocket socket, bool ignoreFirstCancel)
 
             if (_gracefulClose)
             {
-                _application.Input.Complete(error);
+                _application.Input.Complete();
```

Comment:
Why this change?

Reply:
We generally try to not make the write side throw. I noticed I added this during the earlier reconnect work so am just reverting it.

Reply:
We should probably still log something like this though.